src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import FooterLogo from "../../assets/netflix-logo.png";
+import "./Footer.scss";
+
+export default function Footer() {
+    return(
+        <footer className={"footer"}>
```

Comment:
you can avoid brackets if value is string -> `className="footer"` would be enough

Reply:
@lotrien, thanks for the advice, I fixed it